### PR TITLE
New runtime doesn't support prefixing every command

### PR DESCRIPTION
**Pull request by beholders-eye**
`yb build -exec-prefix <something>` isn't working anymore. This tries to fix it.

### CODE REVIEW

File: workspace/build_target.go [hunk 1]
```diff
@@ -156,10 +156,10 @@ func (bt BuildTarget) Build(runtimeCtx *runtime.Runtime, flags BuildFlags, packa
 
 	// Do this after the containers are up
 	for _, envString := range bt.EnvironmentVariables(runtimeCtx.EnvironmentData()) {
-		parts := strings.Split(envString, "=")
+		parts := strings.SplitN(envString, "=", 2)
```
Comment (zombiezen) [inline workspace/build_target.go:159]: Optional: Perhaps this should be extracted out to a function to share with the code in #87?

File: workspace/build_target.go [hunk 2]
```diff
@@ -180,6 +180,10 @@ func (bt BuildTarget) Build(runtimeCtx *runtime.Runtime, flags BuildFlags, packa
 	for _, cmdString := range bt.Commands {
 		var stepError error
 
+		if len(flags.ExecPrefix) > 0 {
+			cmdString = fmt.Sprintf("%s %s", flags.ExecPrefix, cmdString)
```
Comment (zombiezen) [inline workspace/build_target.go:184]: nit for clarity:

```suggestion
		if flags.ExecPrefix != "" {
			cmdString = flags.ExecPrefix + " " + cmdString
```

File: workspace/build_target.go (hunk 1 above)
Comment (beholders-eye) [reply workspace/build_target.go:159]: Makes sense

**Comment by beholders-eye** [comment]
Addressed @zombiezen PR feedback. Thanks!

**Pull request merged**